tasks/systemd_setup.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+- name: install the systemd unit file
+  template:
+    src: 'etc/systemd/system/tinc@.service.j2'
+    dest: '/etc/systemd/system/tinc@.service'
+    owner: 'root'
+    group: 'root'
+    mode: '0644'
+
+- name: autostart tinc
+  service:
+    name: 'tinc@{{ tinc_network }}'
+    enabled: 'yes'
+    state: 'started'
```

Comment:
Verification is needed because this team as its result depends on the availability of systemd.